buildpack.md

```diff
@@ -594,10 +594,17 @@ run-images = ["<run image tag>"]
 ```
 
 Buildpack authors MUST choose a globally unique ID, for example: "io.buildpacks.ruby".
-The buildpack ID MUST contain at least one period (`.`).
+
+The buildpack ID:
+- MUST only contain numbers, letters, and the charactors `.`, `/`, and `-`.
+- MUST NOT be `config` or `app`.
+- MUST NOT be identical to any other buildpack ID when letters in each ID are converted to lower case.
 
 Stack authors MUST choose a globally unique ID, for example: "io.buildpacks.mystack".
-The stack ID MUST contain at least one period (`.`).
+
+The stack ID:
+- MUST only contain numbers, letters, and the charactors `.`, `/`, and `-`.
+- MUST NOT be identical to any other stack ID when letters in each ID are converted to lower case.
```

Comment:
Should this be described as "MUST NOT be identical to any other stack ID when using a case-insensitive comparison"?